### PR TITLE
[IMP] Helpdesk: Reporting 17

### PR DESCRIPTION
**Pull request by jero-odoo**
Docs task: https://www.odoo.com/mail/view?model=project.task&res_id=3914689&access_token=1490b691-cd0d-4a3b-9e70-bdb7df46e425

Updating Helpdesk reporting for V17​, updated content and images

### CODE REVIEW

**Comment by jero-odoo** [comment]
Hey @Felicious this is ready for review. Thanks!

File: content/applications/services/helpdesk/overview/reports.rst
Comment (Felicious) [inline content/applications/services/helpdesk/overview/reports.rst:196]: Here is an opportunity to use the new :icon: to represent this symbol !

File: content/applications/services/helpdesk/overview/reports.rst
Comment (Felicious) [inline content/applications/services/helpdesk/overview/reports.rst:82]:
```suggestion
- :guilabel:`Rating (/5)`: number valued representing customer feedback
```

File: content/applications/services/helpdesk/overview/reports.rst
Comment (Felicious) [inline content/applications/services/helpdesk/overview/reports.rst:203]: This doc is really useful, and it's great that you're fleshing out all of these reports and fields. As a helpdesk beginner though, I'd like to share some of my first impressions.

When looking at a pivot table like the one in the SLA Status Analysis report, I have to admit, I was a bit lost. It might be my lack of industry knowledge, though!

So, maybe in a followup PR, when you get a chance, you could include some example graphs and give us a few pointers on what numbers we should be paying attention to. Like, what's considered a red flag in terms of ticket start/finish times?

Just a thought!

File: content/applications/services/helpdesk/overview/reports.rst
Comment (Felicious) [inline content/applications/services/helpdesk/overview/reports.rst:54]:
```suggestion
- :guilabel:`Rating /5`: number out of five to represent customer feedback (Dissatisfied
```

File: content/applications/services/helpdesk/overview/reports.rst
Comment (Felicious) [inline content/applications/services/helpdesk/overview/reports.rst:8]:
```suggestion
Reports in Odoo *Helpdesk* provide the opportunity to confirm if customer expectations
```

File: content/applications/services/helpdesk/overview/reports.rst
Comment (Felicious) [inline content/applications/services/helpdesk/overview/reports.rst:17]: In the UI, it's spelled "Tickets", but I'm wondering if it's a Belgium mispelling? It doesn't sound like correct grammar, so I'd confirm with someone, or submit a bug report :joy:

**Review by Felicious — APPROVED** [review]
Really clear and well-written doc, @jero-odoo !

It's really nice to finally be able to review one of your docs! :smile: I just had a couple really minor rewording suggestions to consider. Amazing job with this! :heart:

File: content/applications/services/helpdesk/overview/reports.rst
Comment (jero-odoo) [reply content/applications/services/helpdesk/overview/reports.rst:203]: Thats a good point, I do talk a bit about this report in the SLA doc (which I will add a link to) but I can update that section to provide some additional metrics guidelines :+1:

File: content/applications/services/helpdesk/overview/reports.rst
Comment (ksc-odoo) [inline content/applications/services/helpdesk/overview/reports.rst:9]:
```suggestion
Reports in Odoo *Helpdesk* provide the opportunity to manage employee workloads, identify areas for
improvement, and confirm if customer expectations are being met.
```
Optional reworking of the same sentence just to enhance the overall flow of it a bit more. Totally your call, though. 👍

File: content/applications/services/helpdesk/overview/reports.rst
Comment (ksc-odoo) [inline content/applications/services/helpdesk/overview/reports.rst:15]:
```suggestion
reports, go to :menuselection:`Helpdesk app --> Reporting`, and choose to open reports for the
following: :guilabel:`Tickets Analysis`, :guilabel:`SLA Status Analysis`, or :guilabel:`Customer
Ratings`.
```

File: content/applications/services/helpdesk/overview/reports.rst
Comment (ksc-odoo) [inline content/applications/services/helpdesk/overview/reports.rst:23]:
```suggestion
This report is useful for identifying where teams are spending the most time, and helps determine if
```

File: content/applications/services/helpdesk/overview/reports.rst
Comment (ksc-odoo) [inline content/applications/services/helpdesk/overview/reports.rst:40]:
```suggestion
  date. If there is no closed date on the ticket, the current date is used. **This measure is not
```
since the previous definition (and other ones after) seem to use *italics* and not **bold**, do you think changing this last sentence to *italics* would be best to make things appear more uniform? Or, do you think the **bold** formatting is necessary for the message that's being delivered. Just something for you to consider during your revisions. Thanks! 👍

File: content/applications/services/helpdesk/overview/reports.rst
Comment (ksc-odoo) [inline content/applications/services/helpdesk/overview/reports.rst:56]:
```suggestion
- :guilabel:`Remaining Hours on SO`: hours remaining on a linked sales order.
```

File: content/applications/services/helpdesk/overview/reports.rst
Comment (ksc-odoo) [inline content/applications/services/helpdesk/overview/reports.rst:72]:
```suggestion
that have been successful. The results are grouped by teams.
```

File: content/applications/services/helpdesk/overview/reports.rst
Comment (ksc-odoo) [inline content/applications/services/helpdesk/overview/reports.rst:115]:
```suggestion
   On the rating's details page, tick the :guilabel:`Visible Internally Only` checkbox to hide the
```

File: content/applications/services/helpdesk/overview/reports.rst
Comment (ksc-odoo) [inline content/applications/services/helpdesk/overview/reports.rst:141]:
```suggestion
The pivot view can be accessed on any report by selecting the :icon:`oi-view-pivot` :guilabel:`(grid)` icon at the top-right
```

File: content/applications/services/helpdesk/overview/reports.rst
Comment (ksc-odoo) [inline content/applications/services/helpdesk/overview/reports.rst:150]:
```suggestion
click the :icon:`fa-minus-square-o` :guilabel:`(minus)` icon, and de-select the appropriate option.
```

File: content/applications/services/helpdesk/overview/reports.rst
Comment (ksc-odoo) [inline content/applications/services/helpdesk/overview/reports.rst:184]:
```suggestion
   compare data. While viewing either a bar chart or line chart, click the :icon:`fa-database`
```

File: content/applications/services/helpdesk/overview/reports.rst
Comment (ksc-odoo) [inline content/applications/services/helpdesk/overview/reports.rst:185]:
```suggestion
   :guilabel:`(stacked)` icon to toggle the stacked view option on or off.
```

File: content/applications/services/helpdesk/overview/reports.rst
Comment (ksc-odoo) [inline content/applications/services/helpdesk/overview/reports.rst:191]:
```suggestion
filters, without having to reconstruct them every time they are needed.
```

File: content/applications/services/helpdesk/overview/reports.rst
Comment (ksc-odoo) [inline content/applications/services/helpdesk/overview/reports.rst:193]:
```suggestion
To create and save a new *Favorites* configuration on a report, follow the steps below:
```

File: content/applications/services/helpdesk/overview/reports.rst
Comment (ksc-odoo) [inline content/applications/services/helpdesk/overview/reports.rst:203]:
```suggestion
#. Tick the :guilabel:`Shared` checkbox to make this filter configuration available to all other
```

File: content/applications/services/helpdesk/overview/reports.rst
Comment (ksc-odoo) [inline content/applications/services/helpdesk/overview/reports.rst:204]:
```suggestion
   database users. If this checkbox is not ticked, only the user who creates the filter can access it.
```

**Review by ksc-odoo — APPROVED** [review]
Hey, @jero-odoo ! Great work on this PR! Just finished my Final Review, and approving now! 👍 Only have a handful of super-minor suggestions that require your attention. Once all the necessary changes have been implemented, feel free to tag this for Tech Review. Thanks! 🙏

File: content/applications/services/helpdesk/overview/reports.rst
Comment (jero-odoo) [reply content/applications/services/helpdesk/overview/reports.rst:42]: I do want to keep this different just to make sure there is emphasis on when measures are specific to working hours (and therefore follow the working calendar) and when they aren't, since I think that can be a bit confusing for helpdesk.
If there is a ticket that has been "open" for 24 hours, that is tracking time when the team is off the clock. But if you just want to see how long the team is actively working on a ticket, you would select "Hours to close", because that is only going to show the "working hours", so it would not include off work time. (I hope this clears things up).

**Comment by jero-odoo** [comment]
hey @samueljlieber this is ready for tech review. Thanks!

File: content/applications/services/helpdesk/overview/reports.rst
Comment (samueljlieber) [inline content/applications/services/helpdesk/overview/reports.rst:55]: The measure in the UI has parentheses around "/5"
```suggestion
- :guilabel:`Rating (/5)`: number out of five to represent customer feedback (Dissatisfied = 1,
```

File: content/applications/services/helpdesk/overview/reports.rst
Comment (samueljlieber) [inline content/applications/services/helpdesk/overview/reports.rst:160]: The graph view uses the `fa-area-chart` icon, not the `fa-line-chart` :)
```suggestion
Switch to the graph view by selecting the :icon:`fa-area-chart` :guilabel:`(area chart)` icon at the
top-right of the screen. To switch between the different charts, select the *related icon* at the
top-left of the chart, while in graph view.
```

**Review by samueljlieber — APPROVED** [review]
Hi @jero-odoo, your 17.0 improvements to this doc look good to me. Nice use of icons, I found two small corrections, approving and delegating to you!
...
@robodoo delegate=jero-odoo

**Comment by jero-odoo** [comment]
@robodoo r+

**Pull request closed without merging**
